doc/api/errors.md

```diff
@@ -2978,6 +2978,17 @@ import 'package-name'; // supported
 
 `import` with URL schemes other than `file` and `data` is unsupported.
 
+<a id="ERR_UNSUPPORTED_RESOLVE_REQUEST"></a>
+
+### `ERR_UNSUPPORTED_RESOLVE_REQUEST`
+
+An attempt was made to resolve an invalid module referrer. This can happen when
+calling `import()` or `import.meta.resolve()` with either:
+
+* a bare specifier that is not a builtin module from a module whose URL scheme
+  is not `file`.
+* a [relative URL][] from a module whose URL scheme is not a [special scheme][].
```

Comment:
This could probably benefit from an example that would trigger the error.

Reply:
````suggestion
* a [relative URL][] from a module whose URL scheme is not a [special scheme][].

```js
try {
  // Trying to import the package 'bare-specifier' from a `data:` URL module:
  await import("data:text/javascript,import%22bare-specifier%22");
} catch(e) {
  console.log(e.code); // ERR_UNSUPPORTED_RESOLVE_REQUEST
}
```
````